src/bridge/lib/keys.js

```diff
@@ -62,10 +62,14 @@ const genKey = (networkSeed, point, revision) => {
 
 // 'next' refers to setting the next set of keys
 // if false, we use revision - 1
-const attemptSeedDerivation = async (next, args) => {
+const attemptSeedDerivation = async (next, args, assumedRevision) => {
```

Comment:
The fact that this can't reasonable be part of `args` makes me think that's a misleading variable name. Just noting that for the record, no action required here.